beaglebone/imu/src/mag_calibration.txt

```diff
@@ -0,0 +1,3 @@
+-12.72  -7.79  21.22
+0.954  0.021  -0.005 0.021  1.030 -0.009 -0.005  -0.009  1.017
+35.14
```

Comment:
I'd prefer all calibration files be moved to a config file in `config` to avoid random .txt files floating around the repo